<launch>
  
  <!-- EXAMPLE FOR NANO17 -->

  <include file="$(find force_torque_sensor)/launch/ft_sensor_hw.launch.xml">
      <arg name="ip" value="192.168.1.101"/>
      <arg name="name" value="my_sensor" />
      <arg name="type" value="nano17" />
      <arg name="frame_id" value="right_arm_7_link" />
      <arg name="safety_topic" value="/right_arm/emergency_event" />
      <arg name="safety_threshold" value="0.1" />
      <arg name="publish_rate" value="500" />
  </include>
   
</launch>